### PR TITLE
Interruptible copies from cursors

### PR DESCRIPTION
**Pull request by robrix**
We think we’re seeing some copies that we would like to interrupt.

### CODE REVIEW

File: tree-sitter/src/TreeSitter/Cursor.hs [hunk 1]
```diff
@@ -51,4 +51,4 @@ foreign import ccall unsafe "ts_tree_cursor_goto_next_sibling" ts_tree_cursor_go
 foreign import ccall unsafe "ts_tree_cursor_goto_first_child" ts_tree_cursor_goto_first_child :: Ptr Cursor -> IO Bool
 foreign import ccall unsafe "ts_tree_cursor_goto_first_child_for_byte" ts_tree_cursor_goto_first_child_for_byte :: Ptr Cursor -> Word32 -> IO Int64
 
-foreign import ccall unsafe "src/bridge.c ts_tree_cursor_copy_child_nodes" ts_tree_cursor_copy_child_nodes :: Ptr Cursor -> Ptr Node -> IO Word32
+foreign import ccall interruptible "src/bridge.c ts_tree_cursor_copy_child_nodes" ts_tree_cursor_copy_child_nodes :: Ptr Cursor -> Ptr Node -> IO Word32
```
Comment (robrix) [inline tree-sitter/src/TreeSitter/Cursor.hs]: This matches what we do for copies from a node.

**Review by robrix — COMMENTED** [review]
Ready for review.

**Review by tclem — APPROVED** [review]

**Comment by robrix** [comment]
NB: #261 applies to this as well, tho not as badly as to `ts_node_copy_child_nodes`.

**Comment by robrix** [comment]
https://hackage.haskell.org/package/tree-sitter-0.8.0.1

**Pull request merged**